v2/loadbalancers_test.go

```diff
@@ -12,7 +12,8 @@ func TestLoadBalancersCreate(t *testing.T) {
 	c := &mockClient{body: `{ "response": { "loadbalancer":
 	{ "backends": [], "datacenter": "Falkenberg", "frontends": [],
 	"ipaddress": [{"ipaddress": "192.168.0.1", "version": 4}],
-	"name": "myloadbalancer", "loadbalancerid": "lb123456" }}}`}
+	"name": "myloadbalancer", "loadbalancerid": "lb123456",
+	"cost": {"amount": 200, "currency": "SEK", "timeperiod": "month"} }}}`}
```

Comment:
Should probably have a test with amount being a float number as the problem with Euros.